components/form/Input.js

```diff
@@ -20,23 +20,34 @@ const Input = forwardRef(
       }
     };
 
+    const inputClassName =
+      type === "color"
+        ? classNames(
+            disabled
+              ? "bg-primary-low-medium dark:bg-primary-medium-low hover:border-primary-medium-low focus:ring-0 focus:border-primary-medium focus:outline-0 cursor-not-allowed"
+              : "dark:bg-primary-high hover:border-tertiary-medium focus:ring-0 focus:border-tertiary-medium focus:outline-0",
+            "border-2 transition-all duration-250 ease-linear rounded px-1 mb-2 block w-[100px]"
+          )
+        : classNames(
+            disabled
+              ? "bg-primary-low-medium dark:bg-primary-medium-low hover:border-primary-medium-low focus:ring-0 focus:border-primary-medium focus:outline-0 cursor-not-allowed"
+              : "dark:bg-primary-high hover:border-tertiary-medium focus:ring-0 focus:border-tertiary-medium focus:outline-0",
+            "border-2 transition-all duration-250 ease-linear rounded px-6 py-2 mb-2 block w-full"
+          );
```

Comment:
it adds up some more lines if we use switch
```suggestion
let inputClassName = "";
    switch (type) {
      case "color":
        inputClassName = classNames(
          disabled
            ? "bg-primary-low-medium dark:bg-primary-medium-low hover:border-primary-medium-low focus:ring-0 focus:border-primary-medium focus:outline-0 cursor-not-allowed"
            : "dark:bg-primary-high hover:border-tertiary-medium focus:ring-0 focus:border-tertiary-medium focus:outline-0", "border-2 transition-all duration-250 ease-linear rounded px-1 mb-2 block w-[100px]"
        );
        break;
      default:
        inputClassName = classNames(
          disabled
            ? "bg-primary-low-medium dark:bg-primary-medium-low hover:border-primary-medium-low focus:ring-0 focus:border-primary-medium focus:outline-0 cursor-not-allowed"
            : "dark:bg-primary-high hover:border-tertiary-medium focus:ring-0 focus:border-tertiary-medium focus:outline-0", "border-2 transition-all duration-250 ease-linear rounded px-6 py-2 mb-2 block w-full"
        );
    }
```

Reply:
More lines is not a problem, if it improves readability - which I think it does here 👍